python-checks/src/test/resources/checks/verifiedSslTlsCertificateCheck.py

```diff
@@ -34,51 +34,51 @@ def requestsTests():
   # requests-tests.py
   import requests
 
-  requests.request('GET', 'https://example.domain', verify=False) # Noncompliant {{Enable server certificate validation on this SSL/TLS connection.}}
```

Comment:
`requests` is missing from serialization

Reply:
Adding `types-requests` in the requirements helped solved this issue

Reply:
I understood it mypy option `--install-types` should check for type info in typeshed or download types-{libraryName} or get the type information if provided. But [this does not seem to work with requests ](https://stackoverflow.com/questions/65947076/why-is-mypy-finding-no-type-hints-or-library-stubs-for-any-of-my-imports) and maybe other libraries